.github/workflows/image_openeuler.yml

```diff
@@ -84,6 +84,16 @@ jobs:
         username: ${{ vars.QUAY_USERNAME }}
         password: ${{ secrets.QUAY_PASSWORD }}
 
+    - name: Get and modify ref_name
```

Comment:
I prefer to use vLLM v0.8.4 directly in main rather than this PR: https://github.com/vllm-project/vllm-ascend/pull/577

- Consider `main` branch is equal to v0.8.4-dev, so we can just set vLLM to a release version.
This will reduce the risk that main image is OK but v0.8.4 binary is failed. we should release same version image what we validate in CI continuously. 

- For main to main CI, now we have:
https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test.yaml#L92
It's ok to remove this in Dockerfile

- https://vllm-ascend.readthedocs.io/en/v0.8.4rc1/installation.html#setup-vllm-and-vllm-ascend
we will also don't need change the version `Setup vllm and vllm-ascend - Using docker`

Reply:
the only problem then is that there is no main to main image for developers. Is that fine? Any way let'd merge 577 first to unblock release thing